dot/network/service.go

```diff
@@ -384,47 +386,56 @@ func (s *Service) logPeerCount() {
 	}
 }
 
-func (s *Service) publishNetworkTelemetry(done <-chan struct{}) {
+func (s *Service) publishTelemetry(done <-chan struct{}) {
```

Comment:
nit: I usually use `done` to signal when goroutines have exited.  I usually use `cancel` or `stop` to signal a goroutine to shutdown
```suggestion
func (s *Service) publishTelemetry(cancel <-chan struct{}) {
```

Reply:
I agree with Tim, call it `cancel` or `stop`. Or (Tim might hate on me for that), use a context 🤣